pulpcore/pulpcore/app/models/task.py

```diff
@@ -183,6 +186,20 @@ class Task(Model):
     parent = models.ForeignKey("Task", null=True, related_name="spawned_tasks")
     worker = models.ForeignKey("Worker", null=True, related_name="tasks")
 
+    @staticmethod
+    def current():
+        """
+        Returns:
+            pulpcore.app.models.Task: The current task.
+        """
+        try:
+            task_id = current_task.request.id
```

Comment:
(Personal Preference) It would be easier to read if this were `celery.current_task.request.id`, so/from celery.../import celery/